Makefile

```diff
@@ -0,0 +1,70 @@
+ifndef VERBOSE
+.SILENT:
+endif
+
+.PHONY: development
+development:
+	$(eval DEPLOY_ENV=development)
+
+.PHONY: staging
+staging:
+	$(eval DEPLOY_ENV=staging)
+
+.PHONY: sandbox
+sandbox:
+	$(eval DEPLOY_ENV=sandbox)
+
+.PHONY: production
+production:
+	$(eval DEPLOY_ENV=production)
+	$(if $(CONFIRM_PRODUCTION), , $(error Production can only run with CONFIRM_PRODUCTION))
+
+load_domain_config:
+	$(eval include global_config/cpd_ecf_domain.sh)
+
+set-azure-account:
+	echo "Logging on to ${AZ_SUBSCRIPTION}"
+	az account set -s $(AZ_SUBSCRIPTION)
+
+set-azure-resource-group-tags: ##Tags that will be added to resource group on its creation in ARM template
+	$(eval RG_TAGS=$(shell echo '{"Portfolio": "Early Years and Schools Group", "Parent Business":"Teacher Training and Qualifications", "Product" : "Early Careers Framework", "Service Line": "Teaching Workforce", "Service": "Teacher services", "Service Offering": "Manage training for early career teachers", "Environment" : "$(ENV_TAG)"}' | jq . ))
```

Comment:
Should there be an indication in here that ECF is part of CPD?

Reply:
Matt is confirming the tags, we can amend later

Reply:
Updated Porfolio to `Teacher Continuing Professional Development`